src/core/client/admin/test/moderate/regularQueue.spec.tsx

```diff
@@ -813,6 +816,133 @@ it("rejects comment in reported queue", async () => {
   expect(within(reportedCount).getByText("1")).toBeVisible();
 });
 
+it("enables filtering sites by text search", async () => {
+  const commentsResoverStub =
+    createQueryResolverStub<ModerationQueueToCommentsResolver>(
+      ({ variables, callCount }) => {
+        switch (callCount) {
+          case 0:
+            expectAndFail(variables).toEqual({
+              first: 5,
+              orderBy: "CREATED_AT_DESC",
+            });
+            return {
+              edges: [
+                {
+                  node: reportedComments[0],
+                  cursor: reportedComments[0].createdAt,
+                },
+                {
+                  node: reportedComments[1],
+                  cursor: reportedComments[1].createdAt,
+                },
+              ],
+              pageInfo: {
+                endCursor: reportedComments[1].createdAt,
+                hasNextPage: true,
+              },
+            };
+          default:
+            expectAndFail(variables).toEqual({
+              first: 10,
+              query: "second",
+              orderBy: "CREATED_AT_DESC",
+            });
+            return {
+              pageInfo: {
+                endCursor: reportedComments[2].createdAt,
+                hasNextPage: false,
+              },
+            };
+        }
+      }
+    ) as any;
+  const moderationQueuesStub = pureMerge(emptyModerationQueues, {
+    reported: {
+      count: 2,
+      comments: commentsResoverStub,
+    },
+  });
+
+  const sitesResolverStub = createQueryResolverStub<QueryToSitesResolver>(
+    ({ variables, callCount }) => {
+      switch (callCount) {
+        case 0:
+          return siteConnection;
+        default:
+          expectAndFail(variables).toEqual({
+            first: 10,
+            after: null,
+            query: "third",
+          });
+
+          return {
+            edges: [
+              {
+                cursor: "1",
+                node: {
+                  ...sites[0],
+                  id: "site-3",
+                  name: "Third Site",
+                },
+              },
+            ],
+            pageInfo: {
+              hasNextPage: false,
+            },
+          };
+      }
+    }
+  );
+
+  await act(async () => {
+    await createTestRenderer({
+      resolvers: createResolversStub<GQLResolver>({
+        Query: {
+          moderationQueues: () => moderationQueuesStub,
+          sites: sitesResolverStub,
+          settings: () => settingsWithMultisite,
+        },
+      }),
+    });
+  });
+
+  const searchBarContainer = await screen.findByTestId(
+    "moderate-searchBar-container"
+  );
+
+  const siteSelector = await within(searchBarContainer).findByLabelText(
+    "Select site",
+    { exact: false }
+  );
+  expect(siteSelector).toBeInTheDocument();
+
+  userEvent.click(siteSelector);
+
+  await waitFor(() =>
+    expect(within(siteSelector).getByLabelText("Filter results")).toBeVisible()
```

Comment:
It looks like the test is currently failing because you're looking for `Filter results` here but in localization `results` is capitalized (`Filter Results`). Also, you should be able to simplify here to just `expect(await within(siteSelector).findByLabelText("Filter Results")).toBeVisible();` if you like.

Reply:
✅